components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/internal/DCRDataHolder.java

```diff
@@ -99,4 +101,15 @@ public void removeTokenBinder(TokenBinder tokenBinder) {
 
         this.tokenBinders.remove(tokenBinder);
     }
+
+    public ConfigurationManager getConfigurationManager() {
+
+        return configurationManager;
+    }
+
+    public void setConfigurationManager(ConfigurationManager configurationManager) {
+
+        this.configurationManager = configurationManager;
+    }
+
```

Comment:
do we need this new line?

Reply:
Fixed with [108c0c0](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2344/commits/108c0c076e5093d2a8a0f8d17472d0543159554d)